synfig-studio/src/gui/dialogs/canvasproperties.cpp

```diff
@@ -67,7 +67,10 @@ CanvasProperties::CanvasProperties(Gtk::Window& parent,etl::handle<synfigapp::Ca
 	widget_rend_desc.signal_changed().connect(sigc::mem_fun(*this,&studio::CanvasProperties::on_rend_desc_changed));
 
 	Gtk::Alignment *dialogPadding = manage(new Gtk::Alignment(0, 0, 1, 1));
-	dialogPadding->set_padding(12, 12, 12, 12);
+	dialogPadding->set_margin_start(12);
+	dialogPadding->set_margin_end(12);
+	dialogPadding->set_margin_top(12);
+	dialogPadding->set_margin_bottom(12);
```

Comment:
I think using Synfig CSS would be much better since we already have margin rules for all dialogs
https://github.com/synfig/synfig/blob/f9ab46339ebd13f73b947d790617f3197687008e/synfig-studio/src/gui/resources/css/synfig.css#L64-L72
Below is an example of how the above CSS rule is set for our color widget
https://github.com/synfig/synfig/blob/f9ab46339ebd13f73b947d790617f3197687008e/synfig-studio/src/gui/widgets/widget_coloredit.cpp#L348-L349

Reply:
Yes that sounds great.
I searched a bit in the documentation but couldn't find how to apply `"dialog-main-content"` style to `dialogPadding`.

How would you do that?

Reply:
Found why it wasn't working... I'll add a style in the css also for the other asymmetrical layout.